packages/cubejs-athena-driver/src/AthenaDriver.ts

```diff
@@ -63,6 +63,24 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
     this.athena = new Athena(this.config);
   }
 
+  public async tableColumnTypes(table: string) {
```

Comment:
Not 100% why/where the order of columns returned by the source DB matters. Shouldn't Cube column order be defined by the order of measures/dimensions/timeDimensions? Can you please add a small jsdoc comment line to explain /why/ the `ORDER BY columns.column_name` clause is necessary?